### DISCUSSION TITLE
Blacklist For Auto-Raids

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [night/betterttv#6034] [Ideas]
There are some channels that I don't wish to auto-join, but I don't want to outright block all raid-outs.

**Comment by SaphiLC** [comment]
This +10000. It should be easy to implement too since we already have white/black listing of channel chats

**Reply by Duranu23** [reply]
Definitely something that should be implemented, FrankerFaceZ has this abaility but I don't want to use both extensions because I don't know if they will conflict with each other and I don't want to deal with that if they do conflict

**Comment by jrpdevelopment** [comment]
I never thought about BTTV having a Github until I realized there was a feature I wish it had. And then I found this thread. I absolutely LOATHE the "24x7 VOD" channels and wish I could blacklist from joining them, but I definitely want to auto-join raids 99% of the time. I'm sad to see this idea is from 3 years ago and has no traction, but maybe others will agree and it could be implemented? 

Let us blacklist specific channels we don't want to auto-join a raid for please.
